audio_video_recorder/RecordTask[] array
